### PR TITLE
Upgrade to Python 3

### PR DESCRIPTION
**Pull request by giorgiosironi**
No reason to hold it back.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -2,10 +2,10 @@
 set -e
 source prerequisites.sh
 
-if [ ! -e "venv/bin/python2.7" ]; then
-    echo "could not find venv/bin/python2.7, recreating venv"
+if [ ! -e "venv/bin/python3.5" ]; then
```
Comment (de-code) [inline install.sh]: could consider python3.6?

Comment (giorgiosironi) [reply install.sh]: Alas, we don't have it available on Ubuntu LTS in an easy way

Comment (de-code) [reply install.sh]: By Ubuntu LTS you mean an old LTS? It should be easily available on the latest LTS.

Comment (giorgiosironi) [reply install.sh]: 16.04 is the current one we use, [18.04 has some issues] to be solved before we can use it.

**Pull request merged**